src/TesApi.Web/AzureProxy.cs

```diff
@@ -57,34 +57,38 @@ public class AzureProxy : IAzureProxy
         private readonly string azureOfferDurableId;
         private readonly string batchResourceGroupName;
         private readonly string batchAccountName;
-
+        private AzureEnvironment azEnv;
 
         /// <summary>
         /// The constructor
         /// </summary>
+        /// <param name="env">Azure Environment</param>
         /// <param name="batchAccountName">Batch account name</param>
         /// <param name="azureOfferDurableId">Azure offer id</param>
         /// <param name="logger">The logger</param>
-        public AzureProxy(string batchAccountName, string azureOfferDurableId, ILogger logger)
+        public AzureProxy(AzureEnvironment env, string batchAccountName, string azureOfferDurableId, ILogger logger)
```

Comment:
should be renamed to "azEnv" to be consistent with the property

Reply:
Fixed.